### PR TITLE
taxonomy: added Quark subcategory - curd snacks

### PR DESCRIPTION
**Pull request by DanieliusAsm**
### What

- Added new subcategory Curd snacks ([Q2622681](https://www.wikidata.org/wiki/Q2622681))

- Translated, fixed translation for Quarks, curd snacks (Lithuanian)
-  Contributed products to this category [https://world.openfoodfacts.org/category/curd-snacks](https://world.openfoodfacts.org/category/curd-snacks)

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -43772,7 +43772,7 @@ ja:クワルク
 ka:ხაჭო
 ko:크바르크
 lb:Stoffi
-lt:Varškės sūris
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:43775]: Is "Varškės sūris" a correct name that we should add as a synonym?

**Review by stephanegigandet — CHANGES_REQUESTED** [review]
categories.txt.bak was added by mistake I think could you remove it?

File: taxonomies/categories.txt (hunk 1 above)
Comment (DanieliusAsm) [reply taxonomies/categories.txt:43775]: For en:Quarks,Quark it should be lt:Varškės,Varškė

**Comment by DanieliusAsm** [comment]
@stephanegigandet I deleted categories.bak not sure if you noticed it.

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Pull request merged**